### DISCUSSION TITLE
Asynchronicity - support for ascyncio?

### DISCUSSION THREAD

**Discussion by j4hangir** [spyoungtech/ahk#161] [Ideas]
Any chance to have this beautiful library become asyncio-compatible?

**Comment by spyoungtech** [comment]
Yes! This is definitely in the realm of possibility for the future.

**Comment by spyoungtech** [comment]
@j4hangir work on this is now in progress in the asyncio branch

I think asyncio also gives us the path to Python callbacks for hotkeys #9 

Also see some huge performance opportunities here as well, which I hope to share in the future.

**Comment by spyoungtech** [comment]
Asyncio support is now merged and released in version 0.12.0 🎉
